src/Simplex/Chat/Store/NoteFolders.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Store.NoteFolders where
+
+import Control.Monad.Except (ExceptT (..))
+import Control.Monad.IO.Class (liftIO)
+import Data.Time (getCurrentTime)
+import Database.SQLite.Simple (Only (..))
+import Database.SQLite.Simple.QQ (sql)
+import Simplex.Chat.Store.Shared (StoreError (..), checkConstraint)
+import Simplex.Chat.Types (NoteFolder (..), NoteFolderId, NoteFolderName, User (..))
+import Simplex.Messaging.Agent.Protocol (UserId)
+import Simplex.Messaging.Agent.Store.SQLite (firstRow)
+import qualified Simplex.Messaging.Agent.Store.SQLite.DB as DB
+
+createNoteFolder :: DB.Connection -> User -> ExceptT StoreError IO ()
+createNoteFolder db User {userId} =
+  checkConstraint SENoteFolderAlreadyCreated . liftIO $
+    DB.execute db [sql| INSERT INTO note_folders (user_id) VALUES (?) |] (Only userId)
```

Comment:
I think everywhere where it's single line current code uses just quotes, why change it
```suggestion
    DB.execute db "INSERT INTO note_folders (user_id) VALUES (?)" (Only userId)
```